### ISSUE TITLE
Add support for @types/xrm@v8

### ISSUE THREAD

**Issue by BetimBeja** [OliverFlint/XrmTypesGen#20]
There are still a lot of old on premise implementations working with @types/xrm v8, I happen work on a project like this daily 😀
It would be nice to support those projects too.

**Comment by OliverFlint** [comment]
I assume there are some functions/apis that are in the current template that are not available in v8?

**Comment by BetimBeja** [comment]
Yes. There are some api's that are different, and there has been some moving of typings (ex. from `Xrm.Page.BooleanAttribute` in v8 to `Xrm.Attributes.BooleanAttribute` in v9)
I must confess that is has been more than a year that I am not engaged in any v8 maintenance myself, and I don't even have access to such environment to test the tool.

**Comment by OliverFlint** [comment]
ok, I guess we can add and `api-version` switch, and then I should be able to deal with it all in the template.
